Smooch.postman_collection.json

```diff
@@ -2178,7 +2178,7 @@
 								}
 							],
 							"cookie": [],
-							"body": "{\n    \"appUser\": {\n        \"_id\": \"deb920657bbc3adc3fec7963\",\n        \"userId\": \"steveb@channel5.com\",\n        \"givenName\": \"Steve\",\n        \"signedUpAt\": \"2015-10-08T23:52:11.677Z\",\n        \"properties\": {\n          \"favoriteFood\": \"pizza\"\n        },\n        \"conversationStarted\": false,\n        \"credentialRequired\": false,\n        \"clients\": [],\n        \"pendingClients\": [\n            {\n                \"id\": \"d383f9f4-c8d2-42dd-9f7c-f525fad6849d\",\n                \"platform\": \"twilio\",\n                \"displayName\": \"+15145555555\"\n            }\n        ]\n    }\n}"
+							"body": "{\n    \"appUser\": {\n        \"_id\": \"deb920657bbc3adc3fec7963\",\n        \"userId\": \"c7f6e6d6c3a637261bd9656f\",\n        \"givenName\": \"Steve\",\n        \"signedUpAt\": \"2015-10-08T23:52:11.677Z\",\n        \"properties\": {\n          \"favoriteFood\": \"pizza\"\n        },\n        \"conversationStarted\": false,\n        \"credentialRequired\": false,\n        \"clients\": [],\n        \"pendingClients\": [\n            {\n                \"id\": \"d383f9f4-c8d2-42dd-9f7c-f525fad6849d\",\n                \"platform\": \"mailgun\",\n                \"displayName\": \"their@email.com\"\n            }\n        ]\n    }\n}"
```

Comment:
I changed `userId` from `steveb@channel5.com` to `c7f6e6d6c3a637261bd9656f`... is that correct?

Reply:
Under v1 `userId` meant an externally defined id for a user, which could be an email, it's only under v2 do you mean `userId` as something like `c7f6e6d6c3a637261bd9656f`